addons/packages/harbor/2.2.3/README.md

```diff
@@ -18,87 +18,119 @@ The following configuration values can be set to customize the Harbor installati
+
+Please refer the following steps to configure Harbor.
 
 ## Installation
 
 The Harbor package requires use of Contour for ingress, cert-manager for certificate generation and local-path-storage for persistent volume claims (only in case the provider is Docker)
 
 **NOTE**: To install Harbor on Darwin with the provider as Docker, update the `/etc/docker/daemon.json` file with the following and restart Docker.
 
-   ```shell
+   ```sh
    {
       "insecure-registries": ["0.0.0.0/0"]
    }
    ```
 
 1. Install local-path-storage (In case the provider is Docker)
 
-   ```shell
-   tanzu package install local-path-storage -p local-path-storage.community.tanzu.vmware.com -v 0.0.20
+   ```sh
+   tanzu package install local-path-storage \
+      --package-name local-path-storage.community.tanzu.vmware.com \
+      --version ${LOCAL_PATH_STORAGE_PACKAGE_VERSION}
    ```
 
 1. Install the cert-manager package:
 
-   ```shell
-   tanzu package install cert-manager -p cert-manager.community.tanzu.vmware.com -v 1.4.0
+   ```sh
+   tanzu package install cert-manager \
+      --package-name cert-manager.community.tanzu.vmware.com \
+      --version ${CERT_MANAGER_PACKAGE_VERSION}
    ```
 
 1. Install the Contour package using one of the following methods, depending on whether your workload cluster supports Service type LoadBalancer:
 
    If your workload cluster supports Service type LoadBalancer, execute this command:
 
-   ```shell
-   tanzu package install contour -p contour.community.tanzu.vmware.com -v 1.17.1
+   ```sh
+   tanzu package install contour \
+      --package-name contour.community.tanzu.vmware.com \
+      --version ${CONTOUR_PACKAGE_VERSION}
    ```
 
-   or
+   Or
 
    If your workload cluster doesn't support Service type LoadBalancer, use NodePort with hostPorts enabled instead by following these steps:
 
-   1. In `contour.community.tanzu.vmware.com-values.yaml`, set the following parameters:/
-      `envoy.service.type: NodePort`/
-      `envoy.hostPorts.enable: true`
+   1. Set `envoy.service.type: NodePort` and `envoy.hostPorts.enable: true` in `contour-values.yaml`
    1. Run the following command to apply the configuration:
 
       ```sh
-      tanzu package install contour -p contour.community.tanzu.vmware.com -v 1.17.1 --values-file contour-values.yaml
+      tanzu package install contour
+         --package-name contour.community.tanzu.vmware.com \
+         --version ${CONTOUR_PACKAGE_VERSION} \
+         --values-file contour-values.yaml
       ```
 
 1. Configure Harbor Package
 
-   Download the `values.yaml` file from [addons/packages/harbor/2.2.3/bundle/config/values.yaml](https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/harbor/2.2.3/bundle/config/values.yaml) to check all configuration values for Harbor Package.
+   Configure with the `harbor-values.yaml` file you obtained berfore.
 
    Optionally get the helper script for configuring Harbor:
 
-   ```shell
+   ```sh
```

Comment:
actually, the user doesn't have to do `imgpkg pull` again, since the imgpkgBundle bundle has been downloaded into `/tmp/harbor-package-PACKAGE-VERSION`

Reply:
This is a full description for users to refer `imgpkg` in case of they didn't use it before. (cause they can either get values.yaml by downloading or pulling image)